sequencing/vcf_qc/Readme.md

```diff
@@ -1,21 +1,19 @@
-<!-- dx-header -->
 # VCF QC (DNAnexus Platform App)
 
-Calls [GATK ApplyRecalibration](https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_variantrecalibration_VariantRecalibrator.php) and [GATK VariantFiltration](https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_filters_VariantFiltration.php) to apply filters to VCF files.
+## What does this applet do?
 
-This is the source code for an app that runs on the DNAnexus Platform.
-For more information about how to run or modify it, see
-https://wiki.dnanexus.com/.
-<!-- /dx-header -->
+This app uses [GATK ApplyRecalibration](https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_variantrecalibration_VariantRecalibrator.php) and [GATK VariantFiltration](https://www.broadinstitute.org/gatk/guide/tooldocs/org_broadinstitute_gatk_tools_walkers_filters_VariantFiltration.php) to apply filters to VCF files.
 
+## What data are required for this app to run?
 
+This app requires:
+1. The array of VCF (variant calling format) files (`*.vcf.gz`) and their associated index files (`*.vcf.gz.tbi`). VCF is the standard format for variant calling.
+2. The GATK 3 tar ball
+3. The reference genome in FASTA format (`*.fasta` or `*.fasta`), the Picard FASTA index (`*.dict`), and the satools FASTA index (`*.fasta.fai`)
 
-<!--
-TODO: This app directory was automatically generated by dx-app-wizard;
-please edit this Readme.md file to include essential documentation about
-your app that would be helpful to users. (Also see the
-Readme.developer.md.) Once you're done, you can remove these TODO
-comments.
+The user can also provide the BED interval of exon target and other options for variant filtering.
```

Comment:
"Users can also provide a BED interval file containing exon targets **and other options for variant filtering**." (Would the BED file also contain other options for variant filtering?)